src/components/v2/V2Create/tabs/ReviewDeployTab/index.tsx

```diff
@@ -22,7 +22,7 @@ export default function ReviewDeployTab() {
           marginTop: '4rem',
         }}
       >
-        <Space size="large" direction="vertical">
+        <Space size="large" direction="vertical" style={{ width: '100%' }}>
```

Comment:
Somehow this got lost in latest releases. Fixes this:

<img width="1241" alt="Screen Shot 2022-05-06 at 10 37 49 am" src="https://user-images.githubusercontent.com/96150256/167296773-90a5c401-177c-420f-bc99-4ebfacac0962.png">